src/topn.c

```diff
@@ -0,0 +1,126 @@
+#include "data.table.h"
+
+static inline void swap(int *a, int *b) { int tmp=*a; *a=*b; *b=tmp; }
+
+static inline bool icmp(const int *x, const int a, const int b, const bool min, const bool nalast) {
+  if (x[a]==x[b]) return(a > b);
```

Comment:
in C, return is not a function like R, so I think dropping `(`/`)` is good (some linters will require this)